Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -42,6 +42,7 @@ class HTTPClient {
         config.httpMaximumConnectionsPerHost = 1
         config.timeoutIntervalForRequest = requestTimeout
         config.timeoutIntervalForResource = requestTimeout
+        config.urlCache = nil // We implement our own caching with `ETagManager`.
```

Comment:
Hmm I wonder, if this was the case before, did we ever receive 304 responses in our code? As in, would the urlCache always kick in? Not sure how it works internally though.

Reply:
We did, but we didn’t really have any actual coverage for it other that using the HTTP mocks which could have messed with this.